src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -384,6 +384,11 @@
             </span>
           </div>
         </div>
+        <div *ngIf="selectedVersion?.frozen && hasHttpImports" class="pt-3">
+          <mat-card class="alert alert-warning" role="alert">
+            <mat-icon>warning</mat-icon> This version is snapshotted but contains HTTP imports.
```

Comment:
It may not be immediately apparent to all end users why this is warning-worthy. I think you should have a fuller explanation here, and/or Denis' suggested link (and that doc page may need updating too).